app/learning.js

```diff
@@ -1,33 +1,56 @@
 /* file: learning.js
- * authors: H. Bouakaz, S. de Vliet, S. de Jong & E. Werkema
- * date: 5/3/2016
- * version 1.1
+ * authors: H. Bouakaz, S. van Vliet, S. de Jong & E. Werkema
+ * date: 25/3/2016
+ * version 1.2
  *
  * Description:
  * Main script for initiating the learning app.
  */
 
-define(['jquery', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer'], function ($, bootstrap, config, db, messages, question, timer) {
-  timer.startTimer(config.constant("TIME_LIMIT"));
+define(['jquery', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer','app/database'], function ($, bootstrap, config, db, messages, question, timer,db) {
+  timer.startTimer(".timer", config.constant("TIME_LIMIT"));
+  var waitingForEnter = false;
 
   // Disable autocomplete that provides suggestions when typing words
   $('input').attr('autocomplete', 'off');
-
+//when the page is loaded we get the datasetId from the page url and load the dataset from the databese
   $(document).ready(function() {
+    var url =   window.location.href
```

Comment:
Currently, our program uses several webpages, so we use the query string (everything behind the ? in the URL) to transport values from one page to another. Because we only have one simple value, we do it similarly to what is suggested here: http://stackoverflow.com/questions/22094081/how-to-get-url-value-after-question-mark-in-javascript

We have not yet decided whether we will keep our program as multiple webpages, or all on one dynamically changing page. In the latter case, there will be no need for query strings anymore. If, however, we keep using multiple pages, we could adapt a nicer method like this:  https://css-tricks.com/snippets/javascript/get-url-variables/

I've made a Trello card for future consideration here: https://trello.com/c/NbqpFH3i